src/dc_receive_imf.rs

```diff
@@ -1525,7 +1526,7 @@ fn search_chat_ids_by_contact_ids(
                        WHERE cc.chat_id IN(SELECT chat_id FROM chats_contacts WHERE contact_id IN({})) \
                          AND c.type=120 \
                          AND cc.contact_id!=1 \
-                       ORDER BY cc.chat_id, cc.contact_id;",
+                       ORDER BY cc.chat_id, cc.contact_id;", // 1=DC_CONTACT_ID_GROUP
```

Comment:
is DC_CONTACT_ID_GROUP correct here?

Reply:
no, good catch, this is the 120 two lines above